docs/conf/portland/2023/writing-day.rst

```diff
@@ -71,8 +71,26 @@ Our Writing Day goals:
 - `Draft an operating model <https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/operating-model/outline.md>`_ 
   to help internal stakeholders understand how to interact with the dbt Docs team.
 
-Check out dbt's `contribution guidelines <https://github.com/dbt-labs/docs.getdbt.com#writing-content>`_ and - `style guide <https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md>`__.
+Check out dbt's `contribution guidelines <https://github.com/dbt-labs/docs.getdbt.com#writing-content>`_ and `style guide <https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md>`__.
 
+Writerside
+~~~~~~~~~~
+
+`Writerside <https://jb.gg/writerside>`_ is an authoring and publication solution for
+writing technical docs built on the same platform as JetBrains IDEs. It comes with
+a handful of doc templates out of the box. You can use them to author common types
+of content like overviews, how-to guides, and reference material. Our goal is to
+share best practices and help writers who are starting new projects discover
+templates that help them achieve their goals.
+
+We hope to expand our `template library <https://github.com/JetBrains/writerside-project-templates/blob/main/Writerside/topics/Default-topic.md>`_, to include:
```

Comment:
@plaindocs  I found this online, they did not include this link in their original submission. I'm not sure if they are accepting PRs.

Is it okay for them to simply collect template ideas and not have attendees submit to a project on GitHub/GitLab?

Reply:
I don't see why not.